test/spec/reverse_route_spec.coffee

```diff
@@ -0,0 +1,39 @@
+ReverseRoute = require '../../src/reverse_route'
+
+
+describe 'ReverseRoute', ->
+
+  beforeEach ->
+    @route = new ReverseRoute
+      path: '/some/path'
+      method: 'GET'
+
+  it 'exposes path', ->
+    expect(@route.path).to.equal '/some/path'
+
+  it 'exposes method', ->
+    expect(@route.method).to.equal 'GET'
+
+
+  describe 'behaves like a string', ->
```

Comment:
This is what we decided on here:
https://github.com/alexdavid/exprestive/issues/19

It might be best as charlie said to just overload toString and valueOf and if you need to do anything more involved just use `routes.editUser(123).url`